alerts-prod-arena.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: "nais.io/v1"
+kind: "Alert"
+metadata:
+  name: veilarbaktivitet-arena-alarm
+  namespace: pto
+  labels:
+    team: pto
+spec:
+  receivers:
+    slack:
+      channel: '#team_dab_alerts'
+      prependText: '<!here> | '
+  alerts:
+    - alert: Høy feilrate veilarbaktivitet
```

Comment:
Er begge i denne fila arenarelatert? Isåfall burde kanskje navnet på alarmen være noe med det